### ISSUE TITLE
Discover Github hostname from the context to support the Github Enterprise Server

### ISSUE THREAD

**Issue by tvishwanatharkin** [google-github-actions/send-google-chat-webhook#17]
### TL;DR

In the message payload, `click_url` for `Open Workflow` button assumes that the host is always github.com, making it hard for teams using Github Enterprise Server.

### Detailed design

Hardcoding github.com in URL formation of [Open workflow](https://github.com/google-github-actions/send-google-chat-webhook/blob/76f0cca8165d1ed26b001d8d833068509b2cb673/src/main.go#L241) button makes it hard of  of It's always good to discover the github host from the `workflownotification` itself. Github context contains the Host URL in `server_url` field.
This allows teams running the Github Enterprise Server onpremise consume this action.
